changelog/db.changelog-1.1.xml

```diff
@@ -2,11 +2,31 @@
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
   <changeSet author="Paul Andrews" id="1">
     <createTable tableName="application">
-      <column autoIncrement="true" name="applicationId" type="INTEGER">
-        <constraints nullable="false" primaryKey="true" primaryKeyName="applicationId_pkey" />
-      </column>      
-      <column name="reference" type="VARCHAR(255)" />
-    </createTable>
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="applicationId_pkey" />
+            </column>
+            <column name="reference" type="VARCHAR(255)">
```

Comment:
It should be ok to reduce the size to 12 as that would fit the reference e.g. `VV-XXXX-XXXX`